tests/assets/stubs/nidcpower_measurement/nidcpower_measurement_pb2.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: nidcpower_measurement.proto
+"""Generated protocol buffer code."""
+from google.protobuf.internal import builder as _builder
+from google.protobuf import descriptor as _descriptor
+from google.protobuf import descriptor_pool as _descriptor_pool
+from google.protobuf import symbol_database as _symbol_database
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+
+
+
+DESCRIPTOR = _descriptor_pool.Default().AddSerializedFile(b'\n\x1bnidcpower_measurement.proto\x12.ni.measurementlink.measurement.tests.nidcpower\"Q\n\x0e\x43onfigurations\x12\x11\n\tpin_names\x18\x01 \x03(\t\x12\x15\n\rcurrent_limit\x18\x02 \x01(\x01\x12\x15\n\rmulti_session\x18\x03 \x01(\x08\"\xa7\x01\n\x07Outputs\x12\x15\n\rsession_names\x18\x01 \x03(\t\x12\x16\n\x0eresource_names\x18\x02 \x03(\t\x12\x15\n\rchannel_lists\x18\x03 \x03(\t\x12\x1a\n\x12\x63onnected_channels\x18\x04 \x03(\t\x12\x1c\n\x14voltage_measurements\x18\x05 \x03(\x01\x12\x1c\n\x14\x63urrent_measurements\x18\x06 \x03(\x01\x62\x06proto3')
```

Comment:
This is missing "stubs/nidcpower_measurement/", which suggests that you either ran protoc directly or moved files around without regenerating them. Please re-run `poetry run python scripts\generate_grpc_stubs.py` to regenerate all of the codegen for this `.proto` file.

Doing this will also update all of the `.pyi` files based on changes to mypy-protobuf. Please post a separate PR for the `.pyi` changes to existing files.

Also, in another PR, please update `.github/check_nims.yml` to run `generate_grpc_stubs.py` and check for differences, similar to https://github.com/ni/nidaqmx-python/blob/master/.github/workflows/build.yml so that the PR build will check for this situation.

Reply:
> This is missing "stubs/nidcpower_measurement/", which suggests that you either ran protoc directly or moved files around without regenerating them.

Yes, I ran protoc directly in the terminal for this specific proto. Now, I ran `poetry run python scripts\generate_grpc_stubs.py` that updated the codegen for this .proto file. 👍🏼 

> Doing this will also update all of the .pyi files based on changes to mypy-protobuf. Please post a separate PR for the .pyi changes to existing files.
> Also, in another PR, please update .github/check_nims.yml to run generate_grpc_stubs.py and check for differences, similar to https://github.com/ni/nidaqmx-python/blob/master/.github/workflows/build.yml so that the PR build will check for this situation.

Sure, I've added a [task](https://dev.azure.com/ni/DevCentral/_workitems/edit/2571249) to make sure this is done as part of the [#AB2569164](https://dev.azure.com/ni/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/2569164) work item.